src/magnify/apps/core/models.py

```diff
@@ -15,3 +16,98 @@ class Meta:
 
     def __str__(self):
         return self.username
+
+class Meeting(models.Model):
+    """Model for one meeting or a collection of meetings defined reccursively"""
```

Comment:
I'm not sure how all these fields will allow scheduling single meetings or recurring meetings. I would like to be sure it's a good idea so you need to provide 2 things:
- a page in the documentation with example of each use case
- if necessary tests with examples of each use case